mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule AuthPlug.MixProject do
   def project do
     [
       app: :auth_plug,
-      version: "1.3.3",
+      version: "1.4.0",
```

Comment:
I actually think this should be `2.0` given that the way we specify the `auth_url` in the `AUTH_API_KEY` is a breaking change. 💭 
Leaving it as `1.4` because we are basically the only ones _using_ this package right now. 
But if anyone `else` was using it, we'd have to do a proper SemVer release.